doc/interop-test-descriptions.md

```diff
@@ -681,6 +695,14 @@ When the client requests COMPRESSABLE payload, the response includes a payload
 of the size requested containing all zeros and the payload type is
 COMPRESSABLE.
 
+### Echo Status
+[Echo Status]: #echo-status
+When the client sends a response_status in the request payload, the server returns
```

Comment:
`response_status` does not exist in messages.proto. This should specify which messages to look for response_status. What should the behavior be for a StreamingInputCallRequest? Should the server use the last received status?